packages/snap/src/use-cases/AccountUseCases.ts

```diff
@@ -165,11 +165,6 @@ export class AccountUseCases {
     const account = await this.#repository.getByDerivationPath(derivationPath);
     if (account && account.network === network) {
       this.#logger.debug('Account already exists: %s,', account.id);
-      await this.#snapClient.emitAccountCreatedEvent(
-        account,
-        correlationId,
-        accountName,
-      );
```

Comment:
We still have an `emitAccountCreatedEvent` below, when the account got inserted for the first time.